devspace.yaml

```diff
@@ -437,9 +437,24 @@ profiles:
     path: /deployments/name=minio/helm/values/accessKey/password
   - op: remove
     path: /deployments/name=minio/helm/values/secretKey/password
+  # TODO: does this annotation bring anything positive? (cf. https://cert-manager.io/docs/usage/ingress/#optional-configuration)
+  # - op: add
+  #   path: /deployments/name=minio/helm/values/ingress/certManager
+  #   value: true
+  - op: replace
```

Comment:
this section is necessary for the cert-manager; just make sure the `cluster-issuer` is adapted to take whatever name Jelastic will have chosen